storage/container_storage_interface/ephemeral-storage-shared-resource-csi-driver-operator.adoc

```diff
@@ -6,18 +6,14 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-
 [role="_abstract"]
 As a cluster administrator, you can use the Shared Resource CSI Driver in {product-title} to provision inline ephemeral volumes that contain the contents of `Secret` or `ConfigMap` objects. This way, pods and other Kubernetes types that expose volume mounts, and {product-title} Builds can securely use the contents of those objects across potentially any namespace in the cluster. To accomplish this, there are currently two types of shared resources: a `SharedSecret` custom resource for `Secret` objects, and a `SharedConfigMap` custom resource for `ConfigMap` objects.
 
 // The Shared Resource CSI Driver in {product-title}, as opposed to the driver for upstream Kubernetes...
 
-:FeatureName: The Shared Resource CSI Driver
-include::snippets/technology-preview.adoc[leveloffset=+1]
-
 [NOTE]
 ====
-To enable the Shared Resource CSI Driver, you must xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc#nodes-cluster-enabling[enable features using feature gates].
+The Shared Resource CSI Driver feature is now generally available and included in Builds 1.1 or later for OpenShift. To use this feature, ensure you are running Build 1.1 or a more recent version.
```

Comment:
I recommend the opposite here - indicate that this operator+CSI driver is deprecated in OCP 4.17, and will be removed in a future version of OpenShift.